frontend/src/container/QueryBuilder/components/ToolbarActions/LeftToolbarActions.tsx

```diff
@@ -0,0 +1,58 @@
+import './ToolbarActions.styles.scss';
+
+import { Button, Switch, Typography } from 'antd';
+import { Atom, MousePointerSquare, Terminal } from 'lucide-react';
+
+interface LeftToolbarActionsProps {
+	selectedView: string;
+	onToggleHistrogramVisibility: () => void;
+	showHistogram: boolean;
+}
+
+export default function LeftToolbarActions({
+	selectedView,
+	onToggleHistrogramVisibility,
+	showHistogram,
+}: LeftToolbarActionsProps): JSX.Element {
+	return (
+		<div className="left-toolbar">
+			<div className="left-toolbar-query-actions">
+				<Button
+					value="search"
```

Comment:
value isn't required. We need to handle via state

Reply:
yes this will be handled when integrated with QB